packages/compile-solidity/package.json

```diff
@@ -47,7 +47,10 @@
     "glob": "^7.1.6",
     "mocha": "8.1.2",
     "sinon": "^9.0.2",
-    "tmp": "^0.2.1"
+    "tmp": "^0.2.1",
+    "ttypescript": "^1.5.12",
```

Comment:
Note, [ttypescript](https://github.com/cevek/ttypescript/issues/122) maintenance status is a concern

Reply:
Understood, but I'm hoping we can use in this PR and figure out a solution across the board.